__tests__/scope/1.html

```diff
@@ -0,0 +1,36 @@
+<span *transloco="let t; read: 'scope1'">
+  {{ t('1') }}
+</span>
+
+<span *transloco="let t; read: 'scopeAlias2'">
+  {{ t('1') }}
+</span>
+
+<span *transloco="let t; read: 'scope3'">
+  {{ t('1') }}
+</span>
+
+<span *transloco="let t; read: 'scope4'">
+  {{ t('1') }}
+</span>
+
+<span *transloco="let t; read: 'scope5'">
+  {{ t('1') }}
+</span>
+
+<span *transloco="let t; read: 'scopeAlias6'">
+  {{ t('1') }}
+</span>
+
+<span *transloco="let t; read: 'scope7'">
+  {{ t('1') }}
+</span>
+
+<span *transloco="let t; read: 'scope8'">
+  {{ t('1') }}
+</span>
+
+<span *transloco="let t; read: 'scopeAlias9'">
+  {{ t('1') }}
+</span>
```

Comment:
Let's mix some without the read property. (e.g `t('scope8.1')`)
WDYT about adding support for the `prefix` property? should be a small addition